kibana/values.yaml

```diff
@@ -65,7 +65,12 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 1000
 
-serviceAccount: ""
+serviceAccount:
```

Comment:
```
==> Linting ./
[ERROR] templates/: render error in "kibana/templates/serviceaccount.yaml": template: kibana/templates/serviceaccount.yaml:1:14: executing "kibana/templates/serviceaccount.yaml" at <.Values.managedServiceAccount.create>: map has no entry for key "managedServiceAccount"
```

This value is still referring to the previous changes so still needs to be updated for `managedServiceAccount`.